src/patterns/error-validation/examples/errors-proto/errors/index.njk

```diff
@@ -299,6 +305,31 @@ layout: none
                 }
             })
         }}
+
+        {{
+            onsDuration({
+                "label": "Duration",
+                "id": "address-duration",
+                "dontWrap": true,
+                "error": {
+                    "text": "Enter a number that is less that 60",
```

Comment:
```suggestion
                    "text": "Enter a number than is less that 60",
```

Reply:
Im guessing this is an error so I wont commit this one

Reply:
I previously commented on the wrong one, sorry.
```suggestion
                    "text": "Enter a number that is less than 60",
```